src/overlays/actors/ovl_En_Ba/z_en_ba.c

```diff
@@ -66,35 +76,450 @@ static ColliderJntSphInit D_809B80D4 = {
         COLSHAPE_JNTSPH,
     },
     2,
-    D_809B808C,
+    sJntSphElementInit,
+};
+
+void EnBa_SetupAction(EnBa* this, EnBaActionFunc actionFunc) {
+    this->actionFunc = actionFunc;
+}
+
+static Vec3f D_809B80E4 = { 0.01f, 0.01f, 0.01f };
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_S8(naviEnemyId, 21, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 1500, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 2500, ICHAIN_CONTINUE),
+    ICHAIN_F32(targetArrowOffset, 0, ICHAIN_STOP),
 };
```

Comment:
Well, `D_809B80E4` is clearly an inlined assignment, which means `sInitChain` must be in-function static. What do we do about this, given that we've preferred `sInitChain` to be out-of-function?

Reply:
discussed in discord already, but id say its not too important at this stage which we do.
 we'll prop revisit the idea of consistently doing in-function init chains during documentation

Reply:
I discussed this with people a couple of weeks back when I was doing this file. I think we decided that this comes up so rarely that it's not worth moving the ichain into init just to fix the one inline vector here. I don't have strong opinions either way though